### PR TITLE
ENH Use SearchableMultiDropdownField to select Members

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-admin/issues/1618

CI requires https://github.com/silverstripe/silverstripe-framework/pull/11057 to be merged first

### CODE REVIEW

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Don't forget to remove the `$membersMap` variable assignment

**Comment by emteknetnz** [comment]
Have removed $membersMap variable

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
LGTM, but needs a dependency bump for admin and framework.

**Comment by emteknetnz** [comment]
Have updated composer.json

File: composer.json
Comment (ChloeSartorelli) [inline composer.json]:
```suggestion
        "silverstripe/admin": "^2.2",
```

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: composer.json
Comment (emteknetnz) [reply composer.json]: Updated

**Review by GuySartorelli — APPROVED** [review]
LGTM, will merge once admin and framework PRs are merged

**Pull request merged**
